bindings/binding_typescript_wasm/__tests__/__snapshots__/transform.js.snap

```diff
@@ -59,9 +69,13 @@ exports[`transform in strip-only mode should report correct error for unsupporte
   "endLine": 3,
   "filename": "test.ts",
   "message": "TypeScript enum is not supported in strip-only mode",
-  "snippet": "enum Foo {
-                    a, b    
-                    }",
+  "snippet": "   
+     [1:1]
```

Comment:
This should be removed

Reply:
I couldn’t find the option for that

Reply:
I can remove [] and use string search to remove it, though

Reply:
I filed https://github.com/zkat/miette/issues/429

Reply:
Thanks! For the meantime this would work

Reply:
I'll do with a separate PR